.github/workflows/run-pytest-all.yml

```diff
@@ -28,10 +28,10 @@ jobs:
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
```

Comment:
Can you give a brief comment on the version changes, please?

Reply:
Oh, I just happened to see that there was a newer version of this (and of python install) available. I wish I new what is the difference, but I just go by the instinct that newer is better.